ios/Rainbow/AppDelegate.h

```diff
@@ -1,6 +1,4 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
it's okay, but why did you do this?

Reply:
I didn't see why FB should have copyright on this file